src/tests/example.test.ts

```diff
@@ -0,0 +1,11 @@
+function addNumbers(a, b) {
+  return a + b;
+}
+
+describe("math", () => {
+  test("adds 1 + 2 to equal 3", () => {
+    const result = addNumbers(1, 2);
+
+    expect(result).toBe(3);
```

Comment:
verified this myself, checks out

Reply:
Cool, I was about to bust out the TI-89 to verify myself